pages.fr/common/tar.md

```diff
@@ -6,25 +6,29 @@
 
 - Crée une archive à partir de fichiers :
 
-`tar cf {{cible.tar}} {{fichier1 fichier2 fichier3}}`
+`tar cf {{cible.tar}} {{fichier1}} {{fichier2}} {{fichier3}}`
 
-- Crée une archive gzip :
+- Crée une archive gzip à partir de fichiers :
 
-`tar czf {{cible.tar.gz}} {{fichier1 fichier2 fichier3}}`
+`tar czf {{cible.tar.gz}} {{fichier1}} {{fichier2}} {{fichier3}}`
 
-- Extrait une archive (compressée) dans le dossier courant :
+- Crée une archive gzip à partir d'un répertoire en utilisant son chemin relatif :
 
-`tar xf {{source.tar[.gz|.bz2|.xz]}}`
+`tar czf {{target.tar.gz}} --directory={{chemin/vers/répertoire}} .`
 
-- Extrait une archive dans un dossier cible :
+- Extrait une archive (compressée) dans le dossier courant en affichant la liste des fichiers traités :
 
-`tar xf {{source.tar}} -C {{dossier}}`
+`tar xvf {{source.tar[.gz|.bz2|.xz]}}`
```

Comment:
```suggestion
`tar xvf {{chemin/vers/source.tar[.gz|.bz2|.xz]}}`
```

Mandatory fix ([original page](https://github.com/tldr-pages/tldr/blob/main/pages/common/tar.md) must be updated): the same as above.

```suggestion
`tar xvf {{chemin/vers/source.tar.gz|chemin/vers/source.tar.bz2|chemin/vers/source.tar.xz}}`
```

Optional fix ([original page](https://github.com/tldr-pages/tldr/blob/main/pages/common/tar.md) can be updated): "nested" extension list can be unwrapped to placeholder with alternatives.